flutter/gps_pub.dart

```diff
@@ -0,0 +1,5 @@
+final channel = ably.channels.get('driver:elon');
+await channel.publish('position', data: {
+	'position': {'lat': lat, 'long': long},
```

Comment:
Indentation appears to be inconsistent with previous example

Reply:
Given this is in a small code box (width), I think we should default to two spaces i.e. show indentation, but use min whitespace. WDYT?

Reply:
Yes, looks like I missed this in multiple files. 2 spaces gives better readability.
addressed in 98c92021f5d6fcb4cef495127f34d0645a95a6fc